stable/transparent-hugepage/README.md

```diff
@@ -1,31 +1,16 @@
 # Transparent Hugepage Helm Chart
```

Comment:
this chart requires special privileges (the SCC) in OCP. If you are going to mention stuff in the database chart, it should also be mentioned here

Reply:
I'm not sure what "stuff" is. Please be specific or we can hold additional updates for the next PR following this one.

Reply:
you added the "openshift.enabled=true" text. So I assumed that you wanted to have some form of docs in these pages that explain what privileges are required to run these charts. It is not gone from the database chart.

Reply:
The openshift.enable=true text has been removed. We no longer use it in the instructions.